src/components/icons/categories/accommodation.tsx

```diff
@@ -1,15 +1,21 @@
-import React from 'react'
+import type React from "react";
 
-function SvgAccommodation(props: React.SVGAttributes<{}>) {
+function SvgAccommodation(props: React.SVGAttributes<SVGElement>) {
   return (
-    <svg width="1em" height="1em" viewBox="0 0 15 15" {...props}>
+    <svg
+      width="1em"
```

Comment:
This is an example of a change that I applied to all icons I could find. I think this is a sensible default. We can override the `aria-hidden` property in the `props` if necessary.